aws/cloudformation/iam.yml.erb

```diff
@@ -203,6 +203,7 @@ Resources:
             Resource:
             # TODO provide this ARN through a CloudFormation stack output resource reference.
             - !Sub "arn:aws:firehose:${AWS::Region}:${AWS::AccountId}:deliverystream/analysis-events"
+            - !Sub "arn:aws:firehose:${AWS::Region}:${AWS::AccountId}:deliverystream/i18n-string-tracking-events"
```

Comment:
I think this Role resource is used by an old/obsolete Lambda function and doesn't need to be updated, (but wouldn't do any harm to do so either).